code/modules/clothing/suits/moth.dm

```diff
@@ -19,6 +19,26 @@
 
 	create_storage(storage_type = /datum/storage/pockets)
 
+
+/obj/item/clothing/suit/mothcoat/AltClick(mob/user)
+	. = ..()
+	if(iscarbon(user))
+		var/mob/living/carbon/char = user
+		if((char.get_item_by_slot(ITEM_SLOT_NECK) == src) || (char.get_item_by_slot(ITEM_SLOT_OCLOTHING) == src))
+			to_chat(user, span_warning("You can't adjust [src] while wearing it!"))
+			return
```

Comment:
...Ditto all those reviews for this too. Kinda wish these shared a parent type. 
This would also be pretty easy to make as an element, if you wanna learn.

Reply:
Actually Inoticed you copy+pasted this from goliath cloak, do you think you could try making this an element? If you need help I'll be happy to

Reply:
Yeah I would love some help to turn it into an element!

Reply:
Try peeking at other elements first to see the general structure, and if you get stuck feel free to DM me

Reply:
Gotcha